mirrord-agent/src/file.rs

```diff
@@ -76,7 +76,7 @@ impl FileManager {
     pub fn new(pid: Option<u64>) -> Self {
         let root_path = match pid {
             Some(pid) => PathBuf::from("/proc").join(pid.to_string()).join("root"),
-            None => PathBuf::from("/"),
+            None => PathBuf::from("/proc").join("1").join("root"),
```

Comment:
I don't like this - I prefer that when it's ephermal container we just pass pid 1 as the pid..